sdks/python/apache_beam/runners/common.py

```diff
@@ -535,6 +545,17 @@ def __init__(self, window_fn, main_receivers, tagged_receivers):
     self.window_fn = window_fn
     self.main_receivers = main_receivers
     self.tagged_receivers = tagged_receivers
+    # TODO: remove if block after per-element-output-counter released
+    experiments = RuntimeValueProvider.get_value('experiments', str, '')
+    if 'outputs_per_element_counter' in experiments:
+      per_element_output_counter_name = CounterName('per-element-output-count',
+                                                    step_name=operation_name)
+      self.per_element_output_counter = counter_factory.get_counter(
+          per_element_output_counter_name, Counter.DISTRIBUTION)
+    else:
+      self.per_element_output_counter = counter_factory.get_counter(
```

Comment:
If the experiment is disabled, we're still doing all the work, but just not publishing the result?

Reply:
I'll change this part to do nothing.